tests/unit/test_utf8lex_printable_str.c

```diff
@@ -350,6 +350,19 @@ utf8lex_error_t test_utf8lex_printable_str()
         break;
       }
     }
+    if (diff_str == NULL)
+    {
+      if (strlen(expected) < strlen(actual))
+      {
+        int c = strlen(expected);
+        diff_str = &(actual[c]);
+      }
+      else
+      {
+        int c = strlen(actual);
+        diff_str = &(expected[c]);
+      }
+    }
```

Comment:
In cases where the expected is shorter or longer than the actual string for test_printable_str (which takes "\"" and outputs "\\\"" so that a .c file can be generated etc), we now output the characters past the end of the shorter string.